uniflow/flow/config.py

```diff
@@ -217,9 +216,12 @@ class RaterClassificationOpenAIGPT4Config(RaterConfig):
     )
     guided_prompt_template: GuidedPrompt = field(
         default_factory=lambda: GuidedPrompt(
-            instruction="""Rate the answer based on the question and the context.
-        Follow the format of the examples below to include context, question, answer, and label in the response.
-        The response should not include examples in the prompt.""",
+            instruction="""
+            Evaluate the appropriateness of a given answer based on the question and the context.
+            There are few examples below, consisting of context, question, answer, explanation and label.
+            If answer is appropriate, you should give a higher score and vise versa. Check label to score dictionary: {label2score}.
```

Comment:
Out of curiosity, the `score` is never a part of few shot examples below, so does explicitly saying `higher score` and pass in the `label2score` dict helps the performance compared to purely the label? Same for the other cases.

Reply:
Originally, I use this to replace the prompt like "Accept means generated answer is better and reject means worse" to provide more flexibility to users. So they may not change prompt when changing labels (however, examples must be changed). But in my experiments, It did address the problem that sometimes LLM gives contradictory explanation and label when I change 5 labels to 3 labels.

Reply:
![image](https://github.com/CambioML/uniflow/assets/57245896/2d43b4a5-65b1-4986-8590-2cd3ddb050d8)